controller/src/enhancements/player.rs

```diff
@@ -289,6 +289,66 @@ impl Enhancement for PlayerESP {
                     .build();
                 }
             }
+            
```

Comment:
I would propose a different approach to draw the head dot:
1. Check if the head dot mode is not `None`
2. Find the head bone index within the bones Vec ([Iterator::position](https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.position) might come in handy)
3. Calculate the `head_position`
4. Then draw the head dot according to the style (filled or none filled)